server/odc-service/src/main/java/com/oceanbase/odc/service/task/schedule/StdJobScheduler.java

```diff
@@ -173,8 +174,14 @@ private void tryCanceling(Long jobId) throws JobException {
             configuration.getJobDispatcher().stop(JobIdentity.of(jobEntity.getId()));
         } catch (JobException e) {
             log.warn("Stop job occur error: ", e);
-            AlarmUtils.alarm(AlarmEventNames.TASK_CANCELED_FAILED,
-                    MessageFormat.format("Cancel job failed, jobId={0}", jobEntity.getId()));
+            Map<String, String> eventMessage = AlarmHelper.buildAlarmMessageWithJob(jobEntity.getId());
```

Comment:
unnesssary job query via metadb in AlarmHelper,
AlarmHelper may support build event from several scenario, e.g. jobEntity / scheduleEntity / jobContext